xml/api/ticket-2.33.rng

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<grammar xmlns="http://relaxng.org/ns/structure/1.0"
+         datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes">
+
+    <include href="../constraints-3.9.rng">
+        <start combine="choice">
```

Comment:
I don't think `combine="choice"` is necessary. The original `start` component of the `constraints-3.9.rng` schema will be replaced by this one completely, so we end up with only one `start` element. (i.e., it doesn't get combined with anything)

> Define the components of an element to be the children of the element together with the components of any `div` child elements. If the `include` element has a `start` component, then the `grammar` element must have a `start` component. If the `include` element has a `start` component, then all `start` components are removed from the `grammar` element. If the `include` element has a `define` component, then the `grammar` element must have a `define` component with the same name. For every `define` component of the `include` element, all `define` components with the same name are removed from the `grammar` element.

https://relaxng.org/spec-20011203.html#IDAG3YR